elephant/asset.py

```diff
@@ -376,8 +364,9 @@ def intersection_matrix(
     bsts_y = bsts_y.to_sparse_array()
 
     # Compute the number of spikes in each bin, for both time axes
-    spikes_per_bin_x = bsts_x.sum(axis=0)
-    spikes_per_bin_y = bsts_y.sum(axis=0)
+    # 'A1' property returns self as a flattened ndarray.
+    spikes_per_bin_x = bsts_x.sum(axis=0).A1
```

Comment:
maybe `ravel` or `flatten` would be more telling than `A1`? I haven't seen that being used anywhere else.

Reply:
why not learn new things? :)
1. your approach will be lengthy - `np.array(bsts_x.sum(axis=0)).squeeze()`, - because 'flatten()' of a numpy matrix is still a numpy matrix. And a numpy matrix is always 2d (I want 1d).
2. `.A1` property shares the data with the original matrix! No copy is involved.

Reply:
Ok, I see the point. Fine with me to leave it as it is.